src/main.cpp

```diff
@@ -261,6 +261,14 @@ static void set_allowed_options(OptionList *allowed_options)
 			_("Set network port (UDP)"))));
 	allowed_options->insert(std::make_pair("run-unittests", ValueSpec(VALUETYPE_FLAG,
 			_("Run the unit tests and exit"))));
+#ifndef __ANDROID__
+	allowed_options->insert(std::make_pair("render-formspec", ValueSpec(VALUETYPE_FLAG,
+			_("Batch task: render a formspec in file given by --flag, and ouput a screenshot to the path given -o, then exit"))));
```

Comment:
```suggestion
			_("Batch task: render a formspec in file given by --file, and ouput a screenshot to the path given -o, then exit"))));
```

I think this might be a typo.